WordPress/Classes/Utility/Universal Links/Routes+Domains.swift

```diff
@@ -0,0 +1,17 @@
+struct DomainsManageRoute: Route {
```

Comment:
Should this route sit under `Me` routes or should it be separate as is.

Reply:
I don't have a strong opinion on this, but I think the route under `Routes+me.swift` is enough.

```swift
struct MeAllDomainsRoute: Route {
    let path = "/domains/manage"
    let section: DeepLinkSection? = .me
    let action: NavigationAction = MeNavigationAction.allDomains
    let jetpackPowered: Bool = true
}
```